src/modules/displaymanager/main.py

```diff
@@ -487,31 +487,45 @@ def set_autologin(self, username, do_autologin, default_desktop_environment):
             self.root_mount_point, "etc/lightdm/lightdm.conf"
             )
         text = []
+        addseat = False
+        loopcount = 0
 
         if os.path.exists(lightdm_conf_path):
             with open(lightdm_conf_path, 'r') as lightdm_conf:
                 text = lightdm_conf.readlines()
+                # Check to make sure [SeatDefaults] or [Seat:*] is in the config,
+                # otherwise we'll risk malforming the config
+                if not '[SeatDefaults]' in text and not '[Seat:*]' in text:
+                    addseat = True
```

Comment:
`addseat = '[SeatDefaults]' not in text and  '[Seat:*]' not in text` is shorter and more typically Pythonic

Reply:
Cheers, shortened it to that in the code.